proto/ibc/lightclients/wasm/v1/query.proto

```diff
@@ -21,15 +21,11 @@ service Query {
 
 // QueryCodeHashesRequest is the request type for the Query/CodeHashes RPC method.
 message QueryCodeHashesRequest {
-  // pagination defines an optional pagination for the request.
-  cosmos.base.query.v1beta1.PageRequest pagination = 1;
 }
 
 // QueryCodeHashesResponse is the response type for the Query/CodeHashes RPC method.
 message QueryCodeHashesResponse {
   repeated string code_hashes = 1;
-  // pagination defines an optional pagination for the request.
-  cosmos.base.query.v1beta1.PageResponse pagination = 2;
```

Comment:
How come we're removing the page request and page response? Is it unnecessary because we're only using a slice of strings?